soc/arm/st_stm32/stm32f1/dts_fixup.h

```diff
@@ -2,8 +2,6 @@
 
 /* SoC level DTS fixup file */
 
-#define DT_ADC_1_NAME			DT_ST_STM32_ADC_40012400_LABEL
```

Comment:
I'm fine removing this if not needed, but for coherency, I prefer it is removed in all STM32 series as part of a dedicated PR

Reply:
I could do it.